resources/views/Widgets/Common/ImageBoxWidget.twig

```diff
@@ -85,18 +86,22 @@
     {{ Twig.if("imageUrl | trim is not empty") }}
         {% if widget.settings.lazyLoading.mobile %}
             <lazy-img
-                class="img-{{ imageSize }}"
+                picture-class="img-{{ imageSize }}"
                 image-url="{{ Twig.print("imageUrl") }}"
+                fallback-url="{{ fallbackImagePath }}"
                 alt="{{ Twig.print("imageAltText") }}"
                 {{ Twig.if("imageName | trim is not empty") }}title="{{ Twig.print("imageName") }}"{{ Twig.endif() }}>
             </lazy-img>
         {% else %}
-            <img
-                class="img-{{ imageSize }}"
-                src="{{ Twig.print("imageUrl") }}"
-                alt="{{ Twig.print("imageAltText") }}"
-                {{ Twig.if("imageName | trim is not empty") }}title="{{ Twig.print("imageName") }}"{{ Twig.endif() }}
-            />
+            <picture>
+                <source srcset="{{ Twig.print("imageUrl") }}" {{ Twig.if("imageUrl matches '/.?(\.webp)(?:$|\?)/'") }} type="image/webp" {{ Twig.endif() }}>
```

Comment:
```suggestion
                <source srcset="{{ Twig.print("imageUrl") }}" {{ Twig.if("imageUrl matches '/.?(\\.webp)(?:$|\\?)/'") }} type="image/webp" {{ Twig.endif() }}>
```

In php (und damit auch in twig) werden reguläre Ausdrücke nach der PCRE-Syntax angegeben. Hier müssen `\` escaped werden. https://www.php.net/manual/en/reference.pcre.pattern.syntax.php